src/applications/disability-benefits/all-claims/pages/privateMedicalRecords.js

```diff
@@ -101,5 +122,15 @@ export const schema = {
       },
     },
     privateMedicalRecords: attachments,
+    'view:patientAcknowledgement': {
+      type: 'object',
+      required: ['view:acknowledgement'],
+      properties: {
+        'view:acknowledgement': {
```

Comment:
Is it not necessary to capture a user's acknowledgement somewhere? Asking since this will get stripped out of the submit.

Reply:
Oh, good question.

Reply:
Yes, very good question. That checkbox has a special place in 4142 land. The entire 4142a form basically says "I consent to you going to get my records" to the VA. That is this checkbox. So by checking it, they are in fact "Filling out" the 4142a form.

If a Veteran submitted a 4142 form without a 4142a form, their 4142 form would just sit there and not get processed until they submitted that 4142a form. So in the app, they basically fill out that 4142a form, then fill out the 4142 form on the next page. In the end, the 4142 & 4142a forms are generated within 1 PDF.